lasy/backend.py

```diff
@@ -0,0 +1,10 @@
+try:
```

Comment:
Re backend  control: https://github.com/LASY-org/lasy/pull/259#discussion_r1679329582

We could do a similar control as done in matplotlib, e.g.,
```py
import lasy
lasy.use("numpy")  # default: "auto"
```
https://matplotlib.org/stable/users/explain/figure/backends.html

The logic could be, using the usual precedence of options:
- check if this option was set (on the module aka the current process), otherwise
- check env variable to use default, otherwise
- use cupy if found, otherwise
- use numpy